support.md

```diff
@@ -49,23 +49,34 @@ reported issues within 2 business days.
 
 ## Releases
 
-We regularly publish [binary releases of Bazel](https://github.com/bazelbuild/bazel/releases). To
-that end, we announce release candidates on
-[bazel-discuss](https://groups.google.com/forum/#!forum/bazel-discuss); these are binaries that have
-passed all of our unit tests. Over the next few days, we regression test all applicable build
-targets at Google. If you have a critical project using Bazel, we recommend that you establish an
-automated testing process that tracks the current release candidate, and report any regressions.
+We regularly publish [binary releases of Bazel](https://github.com/bazelbuild/bazel/releases).
 
-If no regressions are discovered, we officially release the candidate after a week. However,
+Every beginning of the month (we target the first business day of the month), we create a new release
+candidate for a new MINOR version (e.g. 0.6.0). The work is tracked by a [release bug on GitHub](https://github.com/bazelbuild/bazel/issues?q=is%3Aissue+is%3Aopen+label%3A%22Release+blocker%22+label%3A%22type%3A+process%22) which indicates
+the exact target date for the incoming month. Those release candidate should pass all our unit tests,
```

Comment:
We should file github issues for the next 3-4 releases with planned dates.
For easy discovery, we should either:
  * put the date in bug title
 * or, besides github issues publish release schedule on a separate page.

Also lets add a separate label 'release schedule' that we apply to such issues so that people can easy search for them (`label%3A%22Release+blocker%22+label%3A%22type%3A+process` is unwieldy) @

Reply:
This sounds good, do we need to reflect anything here? I am creating the "release" label simply for release bug.

Reply:
I created the bugs for the rest of the year, let's iterate on the details on the bugs.